atomic_physics/tests/test_mg25_consts.py

```diff
@@ -0,0 +1,60 @@
+"""
+References:
+[1] - R. Srinivas, Laser-free trapped-ion quantum logic with a radiofrequency
+        magnetic field gradient, PhD Thesis, University of Colorado (2016)
+"""
+
+import unittest
+from atomic_physics.ions import mg25
+from atomic_physics.utils import field_insensitive_point
+import numpy as np
+
+S12_41_31_clock_field = 212.8e-4  # [1]
+MW_transition_freqs = {(3,2):1.326456e9,
+                       (2,2):1.460516e9,
+                       (1,2):1.573543e9,
+                       (2,1):1.573432e9,
+                       (1,1):1.686459e9,
+                       (0,1):1.786044e9,
+                       (0,-1):1.975445e9} # [1]
+
+
+class TestMg25Consts(unittest.TestCase):
+    def test_s12_clock(self):
+        """Look for field-insensitive S1/2 3,1 -> S1/2 2,1 transition at 212.8 G
+        """
+        ion = mg25.Mg25(level_filter=[mg25.S12])
+
+        ion.setB(S12_41_31_clock_field[0])
+        l_index = ion.index(mg25.S12, 1, F=3)
+        u_index = ion.index(mg25.S12, 1, F=2)
+
+        # 212.8 G
+        model_field_independent_point = field_insensitive_point(
+            ion, l_index, u_index, B0=S12_41_31_clock_field[0]
+        )
+        self.assertAlmostEqual(
+            model_field_independent_point, S12_41_31_clock_field[0], places=4
```

Comment:
It seems like this might not actually compare to the intended accuracy (see e.g. https://github.com/OxfordIonTrapGroup/atomic_physics/commit/e810a4c809a21f46afd928fe7f5f88afba1e3351).

Reply:
Thank you thats very helpful! I think it should be fixed now